pkg/reconciler/pipelinerun/metrics_test.go

```diff
@@ -101,7 +111,7 @@ func TestRecordPipelineRunDurationCount(t *testing.T) {
 			err = metrics.DurationAndCount(test.taskRun)
 			assertErrIsNil(err, "DurationAndCount recording recording got an error", t)
 			metricstest.CheckDistributionData(t, "pipelinerun_duration_seconds", test.expectedTags, 1, test.expectedDuration, test.expectedDuration)
-			metricstest.CheckCountData(t, "pipelinerun_count", test.expectedTags, test.expectedCount)
+			metricstest.CheckCountData(t, "pipelinerun_count", test.expectedCountTags, test.expectedCount)
```

Comment:
@mattmoor @hrishin wasn't sure about this… I don't fully understand the behavior change there :sweat_smile:

Reply:
This was a while ago, but I think maybe done by @anniefu?

Reply:
Hmm, I created the metricstest helper package a while ago, but didn't do this particular test change. 

A cursory glance implies that someone added a new tag to the "pipelinerun_count" metric so it has a different set of metrics tags as the "pipelinerun_duration_seconds" metric now, which isn't a problem.